app/models/sign_in/access_token.rb

```diff
@@ -102,6 +105,16 @@ def validity_length
       client_config.access_token_duration
     end
 
+    def validate_user_attributes(user_attributes:)
+      validated_user_attributes = {}
```

Comment:
I recommend using `select` here to do this in a more 'ruby' way:

`user_attributes.with_indifference_access.select { |key| client_config.access_token_attributes.include?(key) }`

Reply:
Actually, thinking about solving the string vs symbol part of this, can you create a function in the client config class itself that does this (feel free to change the name of the function):

> def access_token_attributes_array
>   access_token_attributes.map(&:to_sym)
> end

And then you can use the select above in a simpler way that'll return filtered user_attributes with symbols instead of converting them to strings after the filter:
`user_attributes.select { |key| client_config.access_token_attributes.include?(key) }`

Reply:
I went and tested this, and everything is fine if this function returns a hash with string keys, so I'd say forget about the symbol component and just do:
`user_attributes.with_indifferent_access.select { |key| client_config.access_token_attributes.include?(key) }` 

Like I said in the first comment

Reply:
Sorry for the confusion, ha